src/Input/Select/Select.stories.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import Select, { Props as SelectProps } from './Select';
+import { WarningOutlineIcon } from '../../General/Icon/components';
+
+export default {
+  title: 'Input/Select',
+  component: Select,
+  argTypes: {
+    status: {
+      control: null,
+    },
+    error: {
+      control: { type: 'boolean' },
+    },
+  },
+} as Meta;
+
+const Template: Story<SelectProps> = args => (
+  <div style={{ height: '200px' }}>
+    <Select {...args}>
+      <Select.Option value="accountant">Accountant</Select.Option>
+      <Select.Option value="business development">
+        Business Development
+      </Select.Option>
+      <Select.Option value="software engineer">Software Engineer</Select.Option>
+    </Select>
+  </div>
+);
+
+export const DefaultSelect = Template.bind({});
+DefaultSelect.args = {
+  label: 'Jobs',
+};
+
+export const BasicUsage: Story<SelectProps> = () => (
+  <div style={{ height: '200px' }}>
```

Comment:
We should break the repetition here. You can use `BasicUsage.args` to give the template sets of props. See this as an example: https://github.com/glints-dev/glints-aries/pull/528/files#diff-f78cf8d58a1d0fa429b35b62323a518a8c3f579d316777392fb11bdf0dfff5fc